src/Application/test/CalculatorClassGTest.cpp

```diff
@@ -1,5 +1,5 @@
-#include "../inc/Calculator.h"
-#include "../inc/Messages.h"
+#include "Calculator.h"
+#include "Messages.h"
```

Comment:
You renamed Messages to CalculatorStrings, right? Why is it still messages here?